setup.py

```diff
@@ -23,6 +23,7 @@
     "Flask-Cors==3.0.10",
     "flask-swagger==0.2.14",
     "flask-swagger-ui==3.36.0",
+    "flask_caching==1.10.1",
```

Comment:
do we need flask caching? I think we don't, it was just part of a previous iteration of the fixes.